tests/integ/sagemaker/lineage/test_association.py

```diff
@@ -66,7 +66,9 @@ def test_set_tag(association_obj, sagemaker_session):
         if actual_tags:
             break
         time.sleep(1)
-    assert len(actual_tags) == 1
+    # When sagemaker-client-config endpoint-url is passed as argument to hit beta,
```

Comment:
Please remove reference to "beta" and just specify that for some endpoints, the number of tags may be greater than 1

Reply:
Sure.